CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,53 @@
+# Contributor Covenant Code of Conduct
+
+## Overview
+
+Define the code of conduct followed and enforced by the NVFlare project.
+
+### Intended audience
+
+COMMUNITY | DEVELOPERS | PROJECT LEADS
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
```

Comment:
"pledge to making" -> "pledge to make"

"gender identity and expression" missing comma -> "gender identity, and expression"

Reply:
"gender identity and expression" is actually okay here if the expression is in reference to gender